packages/editor/src/components/post-permalink/style.scss

```diff
@@ -9,16 +9,16 @@
 	white-space: nowrap;
 
 	// use opacity to work in various editor styles
-	border: 1px solid $dark-opacity-light-500;
+	border: $border-width solid $dark-opacity-light-500;
 	border-bottom: none;
 	background-clip: padding-box;
 
 	// put toolbar snugly to edge on mobile
-	margin-left: -$block-padding - 1px;	// stack borders
-	margin-right: -$block-padding - 1px;
+	margin-left: -$block-padding - $border-width;	// stack borders
```

Comment:
That's a weird indentation and doesn't really quite explain this, but I think it's saying "This stacks the border on top of another border"?

I dunno, leave it if it's not clear, but since this line is already changing it'd be nice to tidy it up. 🤷‍♂️